spring-cloud-gcp-data-spanner/src/main/java/org/springframework/cloud/gcp/data/spanner/core/mapping/SpannerPersistentEntityImpl.java

```diff
@@ -225,7 +227,9 @@ private void verifyInterleavedProperties() {
 							"The child primary key column (" + childKey.getColumnName()
```

Comment:
Can you just say `"The child primary key column (" +  childEntity.getType().getSimpleName() + "." + childKey.getColumnName() + ") ... does not match that of its parent (" + getType().getSimpleName() + "." + "parentKey.getColumnName() + " ..."`. I think it's easier to read than broken up into two sentences.